content/running_spark_apps_with_emr_on_spot_instances/selecting_instance_types.md

```diff
@@ -58,7 +58,10 @@ r5a.4xlarge
 r5a.xlarge
 r5d.2xlarge
 r5d.4xlarge
-r5d.xlarge         
+r5d.xlarge
+r6i.2xlarge

```

Comment:
remove r6i, EMR has not support for r6i yet. https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-supported-instance-types.html
Let's add above link to the below line - 

* Instances types that are not supported by EMR such as R5N, R5ad and R5b. Enhanced z, I and D Instance families, which are priced higher than R family. So basically, adding a deny list with the regular expression .*n.*|.*ad.*|.*b.*|^[zid].*. You can find the list of supported instance types [here](https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-supported-instance-types.html)